src/main/kotlin/sliv/tool/data_structures/landmarks/PointLandmark.kt

```diff
@@ -0,0 +1,3 @@
+package sliv.tool.data_structures.landmarks
+
+data class PointLandmark(val uid: Long, val position: Point)
```

Comment:
why don't use existing models?

Reply:
We need to create a factory that will create a landmark for a scene from just data classes containing parsed information. In any case, the data is first parsed into data classes corresponding to the format